### ISSUE TITLE
Bullet list in Markdown Front Matter triggers MD032

### ISSUE THREAD

**Issue by thom4parisot** [markdownlint/markdownlint#130]
Example:

``` md

---
title: FooBar
categories:
- '2016'
published: true

---

Lorem ipsum.
```

`mdl` will assume the bullet list in `categories` to infringe the rule `MD032`.
Is it a bug or is there a way to tell mdl to ignore parsing Front Matter?

Thanks a lot

**Comment by DavidAnson** [comment]
FYI, you can see how I handled front matter in the Node.js implementation here:
https://github.com/DavidAnson/markdownlint/commit/a467f8b3e611692dc56c6118971f9e4a30e8295f

You should be able to reuse the regular expression and tests if interested. Hope this helps!

**Comment by thom4parisot** [comment]
Thanks! I have had a look at the codebase (mostly `kramdown.rb` and `doc.rb`) but I cannot figure out where all the magic happens, if I shall delete the front matter block or provide it somewhere else so as new rules could be added in regard.

Any hint to help me figure it out?

Thank you :-)

**Comment by GarthDB** [comment]
Same things with MD034. If I include a url as a property in the front matter it flags it.
